pkg/ee/group-project-binding/controller/doc.go

```diff
@@ -0,0 +1,29 @@
+//go:build ee
+
+/*
+                  Kubermatic Enterprise Read-Only License
+                         Version 1.0 ("KERO-1.0”)
+                     Copyright © 2020 Kubermatic GmbH
```

Comment:
```suggestion
                     Copyright © 2022 Kubermatic GmbH
```
It affects all files